near/omni-bridge/src/lib.rs

```diff
@@ -198,6 +199,13 @@ impl FungibleTokenReceiver for Contract {
     ) -> PromiseOrValue<U128> {
         let parsed_msg: InitTransferMsg = serde_json::from_str(&msg).sdk_expect("ERR_PARSE_MSG");
         let token_id = env::predecessor_account_id();
+
+        if self.acl_has_role(Role::NativeFeeRestricted.into(), sender_id.clone())
```

Comment:
Please merge with main and change the `sender_id` to `signer_id` due to this changes https://github.com/Near-One/omni-bridge/pull/284